tox.ini

```diff
@@ -19,3 +19,28 @@ deps =
     yamllint==1.15.0
 commands =
     bash -c "yamllint eve/main.yml salt/**/*.yaml"
+
+[testenv:ci-tests]
+deps =
+    pytest
+    pytest_bdd
+    testinfra
+    paramiko
+    cryptography==2.4.2
+commands =
+    pytest --ssh-config=some/identity/files --hosts=[list, of, hosts] -k "ci" tests
```

Comment:
Can we pass in `ssh-config` and `hosts` as an environment variable somehow? That'd ease Eve integration, I believe.

Reply:
yeah would make sens, will do it